packages/elements/src/card/index.ts

```diff
@@ -42,31 +45,6 @@ export class Card extends BasicElement {
     return VERSION;
   }
 
-  /**
-   * A `CSSResultGroup` that will be used
-   * to style the host, slotted children
-   * and the internal template of the element.
-   * @returns CSS template
-   */
-  static get styles (): CSSResultGroup {
```

Comment:
Please move it back where it was. I have updated the styling guide: https://confluence.refinitiv.com/display/ELF/Code+Styling+Guide

Reply:
Can you give me a reason why you want to move it back to the top of the file? I think styles should be a sibling with the template because it is related and easy to visible no need to scroll up and down.

Reply:
They were always at the top, but anyway, the reason is that static methods belong to the class instance and it is general convention across many programming languages to keep them at the top of the class.

Reply:
That gets that point, but I think it should be a sibling with a template that makes more benefits for developers. I will move it back for now and we will discuss it with everyone about coding style later in the questions channel on MS Team.  😀